test/fixtures.yml

```diff
@@ -122,6 +122,11 @@ cams:
   username: testintegrationc
   password: password9
 
+card_connect:
```

Comment:
It this is safe to share, please add a comment indicating that. If not, this looks like it might be our actual testing creds.